android/.idea/misc.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="JDK" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
This is a general question for changes under .idea folder. Did you update them by hand or by running some command? I'm trying to understand why these changes happened and needed.

Reply:
They are updated by Gradle automatically after the changes i made on the project. Here it is specifying JDK version to avoid any compatibilty problems in the future.

Reply:
Thanks